Mathlib/MeasureTheory/Integral/Indicator.lean

```diff
@@ -0,0 +1,226 @@
+/-
+Copyright (c) 2023 Kalle Kytölä. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kalle Kytölä
+-/
+import Mathlib.MeasureTheory.Constructions.BorelSpace.Metrizable
+import Mathlib.MeasureTheory.Function.StronglyMeasurable.Basic
+
+/-!
```

Comment:
If this material is put into a new file, I will write a proper module docstring. But I would like to ask the reviewers to also comment on whether these lemmas should be placed in (various) existing files instead.

Reply:
I think I will be able to move these to rather natural choices of existing files. The results of the same flavor will then be somewhat scattered, but they do rely on different definitions (measurability vs a.e.-meaurability, null-measurability, etc.), so that seems ok to me, at least.

Reply:
Yes that sounds good

Reply:
Actually, for the two remaining results in the new file `Indicator.lean`, I could not find natural places. The import-prerequisites are `Mathlib.MeasureTheory.Constructions.BorelSpace.Metrizable` and
`Mathlib.MeasureTheory.Integral.Lebesgue`.

Were there tools for this besides `#find_home`? That tool unfortunately reports `Mathlib.MeasureTheory.Integral.Indicator`, i.e., the new file itself.

Besides these two lemmas, the others are now scattered in different files in a way that I believe is quite natural.

Reply:
Of course one could argue that these lemmas do almost nothing but combine the `tendsto_measure_of_ae_tendsto_indicator` with preservation of measurability in limits. But I think for good API, it is  nicer not to request the user to explicitly provide that trivial preservation of measurability. That is why I think we should have the remaining two lemmas.